Easydict/objc/ViewController/Window/BaseQueryWindow/EZBaseQueryViewController.m

```diff
@@ -116,6 +123,14 @@ - (void)viewWillAppear {
     [super viewWillAppear];
     
     [EZLog logWindowAppear:self.windowType];
+    [self recoverStatus];
```

Comment:
It looks strange, why do we need to call this method every time `viewWillAppear`?

Reply:
What if the current window is pinned, does it cause an exception?